operations/orders.md

```diff
@@ -0,0 +1,148 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Orders
+
+## Get orders
+
+An order represents a single set of items that was ordered by a customer. Each order is uniquely identified and can be associated with an invoice item.
+
+**Note:** This operation needs [Authentication](../guidelines/authentication.md) and supports the following JSON:API features:
+
+- [Relationships](../guidelines/relationships.md) - `invoice`, `customer`, `booking`, `tables` using `include` query parameter.
+- [Sparse fieldsets](../guidelines/sparse-fieldsets.md) - supports all fields of `order` and related resources with `fields` query parameter.
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/orders/{id}`
+
+### Response
+
+```javascript
+{
+  "data": {
+    "id": "2496b6bd-a326-4a5a-95c8-359c943dcee8",
+    "type": "orders",
+    "attributes": {
+      "covers": 1,
+      "tableStatus": "seated",
+      "notes": "Please make it quick",
+      "depositAmount": "20.00",
+      "createdAt": "2021-06-29T08:00:00Z",
+      "updatedAt": "2021-06-29T08:00:00Z"
+    },
+    "relationships": {
+      "invoice": {
+        "data": {
+          "id": "4014e105-57d6-4b12-a4fc-164601ec53e9",
+          "type": "invoices"
+        }
+      },
+      "customer": {
+        "data": {
+          "id": "4014e105-57d6-4b12-a4fc-164601ec53e9",
+          "type": "customers"
+        }
+      },
+      "register": {
+        "data": {
+          "id": "a0312dc4-e0bf-4a27-837d-a6cf41f54464",
+          "type": "bookings"
+        }
+      },
+      "invoiceItems": {
+        "data": [
+          {
+            "id": "dcd9718c-abb0-4ba2-83c5-ae01505a5391",
+            "type": "tables"
+          }
+        ]
+      }
+    },
+    "links": {
+      "self": "https://pos.mews-demo/com/api/v2/orders/2496b6bd-a326-4a5a-95c8-359c943dcee8"
+    }
+  }
+}
+```
+Below is a list of all possible fields this endpoint can return including relationships fields fetched with include query parameter.
```

Comment:
I'm having trouble correlating the example JSON with the table data. Is this correct? For example, `covers` is mentioned in `attributes` but I can't see a definition anywhere, same for `tableStatus`.

Reply:
I'm wondering if something is off with the documentation generator, it would seem that this entire file makes very little sense... Investigating...

Reply:
I fixed this, we were missing some generator specific code in the OpenAPI definition.